src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        supplier.getRandomColor();
+        int random = new Random().nextInt(100);
+        return "Color: "
+                + supplier.getRandomColor().toString()
+                + ", "
+                + "Number: "
+                + String.valueOf(random);

```

Comment:
let's make ColorSupplier and Rundom class fields. I also noticed that you call the getRandomColor method twice - this is not very correct. And as I previously pointed out, with enum it is better to use the name() method instead of toString(). Also, let's make the number 100 a constant